addon/mixins/prop-types.js

```diff
@@ -60,7 +60,7 @@ export default Ember.Mixin.create({
       const presentPropKeys = Object.keys(this)
       const defaultProps = this.getDefaultProps()
 
-      const needsDefaultProp = Object.assign({}, ...(Object.keys(defaultProps).forEach((key) => {
+      const needsDefaultProp = Object.assign({}, ...(Object.keys(defaultProps).map((key) => {
```

Comment:
Can you also change this to `Ember.assign` as `Object.assign` doesn't work in the version of Firefox most of our CI builds run in.